src/components/shared/MarkerTooltipContents.js

```diff
@@ -17,18 +17,41 @@ function _markerDetail<T>(
   value: T,
```

Comment:
Hi @julienw,

How do I tell flow that I don't want this parameter to accept undefined values, even though I want it to be polymorphic?

Reply:
It's a good question.

I think you can guard the generic type using a `NotVoid` type, like this:
```javascript
type NotVoid = 
  | number
  | string
  | boolean
  | null
  | Array<any>
  | Object;

function _markerDetail<T: NotVoid>(
  ...
```

Now, is it worth it ? :)
[See the live example too.](https://flow.org/try/#0C4TwDgpgBAcg9sAanAlgEygXigKClAHygDsBXAWwCMIAnPQqAZ2BpWIHN6jK44AbCAENiXEqT59RAQRo1BIADzCQAPlEB5SgCsIAY2ABuHDgBmpYvpRxiUMxYUAVAFywEydCoAUAN0F9SEC4OAJRBUADe9DQQwKQ0Nr7+EEYAvsZ2up4AjMFGGZ4ARMysHAW5puaZ4ba8LgDklII0dVAp5flkEu2VngDaWQA0UABMALrdFp7maBAmbBBoE5nlQA)

Reply:
It's clumbsy but it'll help prevent an error I noticed.  Thanks.